src/Rest/Routes/SubmitGeolocationRoute.php

```diff
@@ -101,19 +101,20 @@ public function routeCallback(WP_REST_Request $request)
 		}
 
 		try {
-			$params = $request->get_body();
+			$params = $this->prepareSimpleApiParams($request);
 
-			if (!\is_string($params)) {
+			$data = $params['data'] ?? '';
+			if (!\is_string($data)) {
 				return \rest_ensure_response(
 					$this->getApiErrorOutput(
-						\esc_html__('The geolocation data is malformed or not valid.', 'eightshift-forms'),
+						\esc_html__('The geolocation data is malformed or none valid.', 'eightshift-forms'),
```

Comment:
Was this wording change intentional? Seems weird :)

Reply:
this was wrong merge conflict, fixed